Absolute_Integrator/file_readers/dm3lib_v099.py

```diff
@@ -0,0 +1,638 @@
+#!/usr/bin/env python
+# -*- coding: iso-8859-1 -*-
+
+
+################################################################################
+## Python script for parsing GATAN DM3 (DigitalMicrograph) files
+## and extracting various metadata
+## --
+## warning: *tested on single-image files only*
+## --
+## based on the DM3_Reader plug-in (v 1.3.4) for ImageJ by Greg Jefferis <jefferis@stanford.edu>
+## http://rsb.info.nih.gov/ij/plugins/DM3_Reader.html
+## --
+## Python adaptation: Pierre-Ivan Raynal <raynal@med.univ-tours.fr>
+## http://microscopies.med.univ-tours.fr/
+################################################################################
+
+import sys, os, time
+import struct
+from PIL import Image
+import numpy
+import scipy.misc
+
+__all__ = ["DM3","version"]
+
+version='0.99beta'
```

Comment:
Generally, files themselves are not versioned, but rather inherit the version of the project they're in.  I would also remove the v099 from the filename.